go/vt/topo/test/testing.go

```diff
@@ -40,94 +40,113 @@ func newKeyRange(value string) *topodatapb.KeyRange {
 	return result
 }
 
+func executeTestSuite(f func(*testing.T, *topo.Server), t *testing.T, ts *topo.Server, ignoreList []string, name string) {
+	for _, n := range ignoreList {
+		if n == name {
+			return
```

Comment:
IMO, it's worth logging that it's being skipped to prevent confusion at higher levels.